src/components/Description/Paragraph/index.tsx

```diff
@@ -20,11 +20,13 @@ const CITATIONS_REGEX = `(\\[(${CITATION_REGEX})+\\])`;
 const REMOVE_TRAILING_COMMA_REGEX = /,\s*(?=\])/;
 
 const xReferenceURL = {
-  cazy: 'http://www.cazy.org/fam/{}.html',
-  cog: 'https://ftp.ncbi.nih.gov/pub/COG/COG2014/static/byCOG/{}.html',
-  intenz: 'http://www.ebi.ac.uk/intenz/query?cmd=SearchEC&ec={}',
-  genprop: 'https://www.ebi.ac.uk/interpro/genomeproperties/#{}',
+  cazy: 'http://www.cazy.org/{}.html',
+  cog: 'https://ftp.ncbi.nih.gov/pub/COG/COG2020/data/',
+  intenz: 'https://enzyme.expasy.org/EC/{}',
```

Comment:
All `[intenz:...]` cross-references in abstracts have been replaced by `[ec:...]`

```suggestion
  ec: 'https://enzyme.expasy.org/EC/{}',
```